graph/src/util/stable_hash_glue.rs

```diff
@@ -0,0 +1,82 @@
+use stable_hash::{StableHash, StableHasher};
+use stable_hash_legacy::prelude::{
+    StableHash as StableHashLegacy, StableHasher as StableHasherLegacy,
+};
+
+/// Implements StableHash and StableHashLegacy. This macro supports two forms:
+/// Struct { field1, field2, ... } and Tuple(transparent). Each field supports
+/// an optional modifier. For example: Tuple(transparent: AsBytes)
+#[macro_export]
+macro_rules! _impl_stable_hash {
```

Comment:
would it be worth adding some tests to ensure changes to this code don't break anything?

Reply:
Yes, I can add some backwards-compatibility checks for the PoI in particular.

Reply:
This is done.